assets/wizards/popups/views/analytics/index.js

```diff
@@ -1,16 +1,87 @@
 /**
- * Pop-ups wizard, Analytics screen.
+ * External dependencies.
  */
+import { stringify } from 'qs';
+import classnames from 'classnames';
 
 /**
- * Internal dependencies
+ * WordPress dependencies.
  */
-import { withWizardScreen } from '../../../../components/src';
+import { Spinner } from '@wordpress/components';
+import { useEffect, useState } from '@wordpress/element';
+import apiFetch from '@wordpress/api-fetch';
 
 /**
- * Analytics Screen
+ * Internal dependencies.
  */
+import { withWizardScreen, Notice } from '../../../../components/src';
+import Filters from './Filters';
+import Chart from './Chart';
+import Info from './Info';
+import './style.scss';
+import { useFiltersState, useAnalyticsState } from './utils';
 
-export default withWizardScreen( () => {
-	return <p>TK</p>;
-} );
+/**
+ * Popups Analytics screen.
+ */
+const PopupAnalytics = () => {
+	const [ hasFetchedOnce, setHasFetchedOnce ] = useState( false );
```

Comment:
Good idea, since a reduced is used anyway! Changed in 4f9628566fce7f104d619718379ae3819ec9a3d2, but then that changed in be937c04a464160a642a0079cbe9e5760dd79194

Reply:
The reducers are a really nice part of this PR!